synapse/utils/stormcov/plugin.py

```diff
@@ -112,6 +112,9 @@ def dynamic_source_filename(self, filename, frame, force=False):
         nodeid = id(node)
 
         info = self.node_map.get(nodeid, s_common.novalu)
+        if info is None:
```

Comment:
Should this check be fore novalu or am i missing something?

Reply:
this checks for novalu next, but if it is None we can skip it